### ISSUE TITLE
Splashscreen Bypass Patched

### ISSUE THREAD

**Issue by pwnf** [Octoberfest7/TeamsPhisher#34]
Looks like the splash screen bypass has once again been patched by Microsoft reducing the effectiveness of TeamsPhisher. 

Now when you remove the victim from the group they are only able to preview the message and the button for clicking through the splash screen is greyed out. 

Given reliance is once again on the user to click through the splash screen and they can't be removed - I do wonder if its better to revert TeamsPhisher back to just messaging a user without adding them to a group at all.

**Comment by Octoberfest7** [comment]
There is a 'revert_bypass' branch I threw together for this purpose a bit ago. I'll merge it in eventually, along with addressing outstanding pull requests, hopefully in the near future after some other work is complete.

**Comment by er4z0r** [comment]
So from looking at the code in the branch, this just removes the code that removes the user after sending the message.

Does this basically mean that currently there is no (public) bypass for the the "external user" warning?

**Comment by Octoberfest7** [comment]
That is correct, the revert_bypass branch simply makes it so targets can choose to view the message again, whereas in the main branch because the user is removed they are unable to interact with the message. I haven't merged it into main because I lack another account to test with.
